app/api/modules.py

```diff
@@ -1,11 +1,31 @@
-from flask_rest_jsonapi import ResourceDetail
+from flask_rest_jsonapi import ResourceDetail, ResourceList
 
 from app.api.bootstrap import api
 from app.api.schema.modules import ModuleSchema
 from app.models import db
 from app.models.module import Module
 
 
+class ModuleList(ResourceList):
+    """
+    module list
+    """
+    def before_get(self, args, kwargs):
```

Comment:
Should this be `def before_get(self, *args, **kwargs):`?

Reply:
Done @schedutron